controls/3.6/def.cf

```diff
@@ -88,6 +88,9 @@ bundle common def
                            # Assume /16 LAN clients to start with
                            "$(sys.policy_hub)/16",
 
+                           # Uncomment below if HA is used
+                           #"@(def.policy_servers)"
```

Comment:
It might cause cf_null to be in there so I think it's probably safest to keep it commented as suggested.